ctapipe/core/tool.py

```diff
@@ -109,21 +109,33 @@ def main():
             "command-line parameters"
         ),
     ).tag(config=True)
-    log_format = Unicode(
-        "%(levelname)s [%(name)s] (%(module)s/%(funcName)s): %(message)s",
-        help="The Logging format template",
-    ).tag(config=True)
 
-    _log_formatter_cls = ColoredFormatter
+    log_level = Enum(
```

Comment:
I think this is already there in the base class, no?

Reply:
ah, yes it is